Makefile.toml

```diff
@@ -637,10 +637,25 @@ extend = "slt"
 args = ["${@}", "./e2e_test/batch/**/*.slt"]
 description = "Run all batch e2e tests"
 
+[tasks.slt-generated]
+category = "RiseDev - SQLLogicTest"
+extend = "slt"
+args = ["${@}", "./e2e_test/generated/**/*.slt"]
+description = "Run all generated e2e tests"
+
```

Comment:
```suggestion
dependencies = ["docslt"]
```
🤔

Reply:
And `docslt` should be added to somewhere in CI? Maybe `risedev check?`

Reply:
After some simple discussion with @st1page, we think there can be a GitHub workflow like the license checker, that can automatically commit generated SLT files if developers didn't generate them locally.

Reply:
Also thinking about another solution, i.e. to quickly scan all  Rust files for ````slt` to find possible crates that contains SLT blocks at the very beginning of `docslt` command, so that this process can be added in `./risedev check`.

Reply:
Also thinking about another solution, i.e. to quickly scan all  Rust files for ````slt` to find crates that possibly contains SLT blocks at the very beginning of `docslt` command, so that this process can be added in `./risedev check`.

Reply:
You mean make `-p all` faster by filtering out some crates? I think we can simply include only `expr`, `frontend`, `batch`, `stream` by hand 🤣

But I'm afraid it will still take much time though. Ideally we should filter source files, but `rustdoc` won't work then.

Reply:
Even only include `common`, `expr`, `frontend`, `batch` and `stream`, `rustdoc` still takes too much time (18 seconds on my machine) to be added in `check`. But I think it's feasible to ignore generated file in Git, and run `docslt` in CI before running `slt-generated`.